SceneLoader/readme.txt

```diff
@@ -0,0 +1,23 @@
+========================================================================
+    C++/WinRT SceneLoaderComponent Project Overview
```

Comment:
You should convert all readme's to readme.md so that GitHub displays them in a pretty way.